app/assets/javascripts/discourse/app/lib/composer/toolbar.js

```diff
@@ -40,13 +87,15 @@ export class ToolbarBase {
    * @param {string=} buttonAttrs.shortcut
    * @param {boolean=} buttonAttrs.unshift
    * @param {boolean=} buttonAttrs.disabled
+   * @param {Function=} buttonAttrs.active callback function that receives state and returns boolean
    */
   addButton(buttonAttrs) {
-    const g = this.groups.findBy("group", buttonAttrs.group || DEFAULT_GROUP);
+    const g = this.groups.find(
```

Comment:
```suggestion
    const group = this.groups.find(
```

I know this is old code but the `g` is just annoying, IMO 1 letter vars should only be used in a for loop